.gitmodules

```diff
@@ -58,3 +58,6 @@
 [submodule "src/rapidjson"]
 	path = src/rapidjson
 	url = https://github.com/ceph/rapidjson
+[submodule "src/asio"]
+	path = src/asio
+	url = https://github.com/chriskohlhoff/asio.git
```

Comment:
better integration with the standard library (esp. `std::error_code`) is a cool feature, but i worry that standalone asio won't interop with beast - which would be a hard requirement for radosgw